icu4j/main/classes/core/src/com/ibm/icu/impl/SimpleFilteredSentenceBreakIterator.java

```diff
@@ -72,28 +72,22 @@ private final boolean breakExceptionAt(int n) {
         backwardsTrie.reset();
         int uch;
 
-
-
         // Assume a space is following the '.' (so we handle the case: "Mr. /Brown")
         if ((uch = text.previousCodePoint()) == ' ') { // TODO: skip a class of chars here??
             // TODO only do this the 1st time?
         } else {
             uch = text.nextCodePoint();
         }
 
```

Comment:
Up in the constructor, lines 46 and 47, do we need to clone the CharTries? Or maybe it could be done in the clone() function. Or, given the timing, defer the problem? It's a bug, but not a regression.

Reply:
looks like it should be cloned

Reply:
Multiple instances must be independent of each other.

Either clone() needs to clone all of its fields (which I have done now), or else the worker function needs to clone the tries (more memory allocations), or the worker functions needs to create new trie iterator objects for the raw data (also more memory allocations).

I left TODOs for equals() and hashCode() not having meaningful implementations.